### ISSUE TITLE
Default permutation method in genomic_converter() is null, not "rf"

### ISSUE THREAD

**Issue by IdoBar** [thierrygosselin/radiator#4]
According to the `genomic_convereter()` documentation, the default method for imputation is "rf", while in fact it is `NULL`. It caused me some confusion trying to understand where the imputed results are.

**Comment by IdoBar** [comment]
Seemed to be fixed in the recent version (v0.0.4)

**Comment by IdoBar** [comment]
Still appears that way in the detailed imputation.method help: 
> Default: imputation.method = "rf".

**Comment by thierrygosselin** [comment]
I see the problem, it's reusing the imputation module
For it to work seamlessly I used the `rf` as default.
However, for `radiator::genomic_converter` I don't want to use it by default,
And the problem is that i'm using `@inheritParams radiator_imputations_module` so that I don't have to write a new `@param` description to avoid repetition. This is already a problem of circular dependencies... I've extracted the imputation module and made `grur`. 
But `grur` needs `radiator` to import different genomic file format. 
So to circumvent the problem I had to include the imputation module of `grur` inside `radiator`
as an internal function not available for users.

I'll figure out a way to fix this in the doc.

Thanks for reporting this.
